docs/modeling/bounding-boxes.rst

```diff
@@ -0,0 +1,149 @@
+.. _bounding-boxes:
+
+Efficient Model Rendering with Bounding Boxes 
+=============================================
+
+.. versionadded:: 1.1
+
+All `astropy.modeling.Model` subclasses have a ``bounding_box`` attribute that 
+can be used to set the limits over which the model is significant. This greatly 
+improves the effciency of evaluation when the input range is much larger than 
+the characteristic width of the model itself. For example, to create a sky model 
+image from a large survey catalog, each source should only be evaluated over the 
+pixels to which it contributes a significant amount of flux. This task can 
+otherwise be computationally prohibitive on an average CPU. 
+
+The `astropy.modeling.render_model` function can be used to evaluate a model on 
+an input array, or coordinates, limiting the evaluation to the ``bounding_box`` 
+region if it is set. This function will also produce postage stamp images of the
+model if no other input array is passed. To instead extract postage
+stamps from the data array itself, see :ref:`cutout_images`. 
+
+Using the Bounding Box
+----------------------- 
+
+For basic usage, see `astropy.modeling.Model.bounding_box`.
+By default no bounding box is set (``bounding_box`` is `None`), except for 
+individual model subclasses that have a ``bounding_box_default`` function 
+defined. ``bounding_box_default`` returns the minimum rectangular region 
+symmetric about the position that fully contains the model if the model has a
+finite extent. If a model does not have a finite extent, the choice for the 
+``bounding_box_default`` limits is noted in the docstring. For example, see
+`astropy.modeling.functional_models.Gaussian2D.bounding_box_default`.
```

Comment:
This should be shortened like:
```
`Gaussian2D.bounding_box_default <astropy.modeling.Gaussian2D.bounding_box_default>`
```
That reminds me also, this is another good reason to have a default (NotImplementedError) implementation of `bounding_box_default` on the base class, so that there is always a docstring for it.